src/mixins/filing-template-mixin.ts

```diff
@@ -638,30 +638,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setOfficeAddresses(entitySnapshot.addresses)
 
     // Store people and roles **from snapshot** (because we don't change people and roles in an alteration)
-    this.setPeopleAndRoles(
-      entitySnapshot.orgPersons?.map(director => {
```

Comment:
This is no longer needed here as it's now done in new Legal API Mixin method `fetchDirectors()`.